src/app/lib/pageStyles/styledPageDiv.js

```diff
@@ -0,0 +1,46 @@
+import {
```

Comment:
I'm not sure about the naming of this component, `styledPageDiv` - what is it? It is only used on index pages?

Reply:
These styles come from what we had on the Front Page https://github.com/bbc/simorgh/pull/6662/files#diff-2a790056302ef5d6fa004eb0da022458R22, and since the IDX page has also the same, I believe @Bopchy wanted to abstract them to a common file so they can be reused.

Reply:
We could call it `PageContainer` or `PageContent`

Reply:
No Tom, it is also used on frontpages here https://github.com/bbc/simorgh/blob/4e03acc05b07ac95482b35bf5fb6054fdbf31a7f/src/app/pages/FrontPage/index.jsx#L22

Reply:
Sorry - by `index` I meant front pages, IDX pages, FIX pages (slightly confusing terminology, sorry!!!) - just thinking that none of the other pages are going to use this component. What do people think about keeping this colocated with the front page for now, but then exporting it and using it in the IDX page? Then when we're done with this page and moving onto FIX pages, we can see if there's a refactor we can do to pull out the common components.

Reply:
I personally like to keep in a separate file and consume it where it is needed otherwise, we are going to have similar or replicated styles in the different pages and will be more difficult to maintain.

I've actually done something similar on my H1 PR because the Most Read page also uses some of those styles https://github.com/bbc/simorgh/pull/6673/files#diff-40aa0d09e27f268691d9fb22aa66b793R5

Reply:
IMO, this is already a common component between Front pages and IDX pages, so I'd say we can keep it as a separate file for now and see if a refactor will be necessary when we start working on FIX pages

Reply:
OK cool - let's rename it and keep an eye on what we are sharing between front pages/IDX/FIX pages going forward.